addons/project_todo/tests/test_todo_ui.py

```diff
@@ -17,6 +17,9 @@ def test_tour_project_task_activities_split(self):
                 - activities linked to records with either project_id set or
                   linked to a parent task are listed in the 'Task' category
         """
+
+        self.env.user.tz = "UTC"
+
```

Comment:
I don't see why we need to add blank lines around `self.env.user.tz = "UTC"`. 🧐 

```suggestion
        self.env.user.tz = "UTC"
```